tests/test_planet_routes.py

```diff
@@ -0,0 +1,69 @@
+def test_get_all_planets_with_no_records(client):
```

Comment:
Thinking about our routes vs our tests, what gaps do you see in the test suite? What functions could we add tests for to get better test coverage for the project?